Automattic/jetpack
# SubmitButton: Remove now its been replaced with shared Button block

Fixes #15715

#### Changes proposed in this Pull Request:
* Now that blocks using the SubmitButton have been updated to use the shared Button block, the SubmitButton component can be removed as it is now unused.

#### Notes
* The diff associated with this PR D46719-code also removes the `jetpack-contact-form.js` file that was removed in https://github.com/Automattic/jetpack/pull/15362 but left on wpcom
* There is still a pending wpcom diff where the Recurring Payments use of Submit Button was updated ( D43891-code ).

#### Does this pull request change what data or activity we track or use?
No, this will not change any data or activity we track.

#### Testing instructions:
* Apply this patch
* Edit a post and test any blocks that previously used the SubmitButton e.g. Calendly, Mailchimp etc.
* A search of the code base for the component being used should not end with any results.

#### Proposed changelog entry for your changes:

* Removed unused shared SubmitButton component.
